### ISSUE TITLE
Ban Appeal: 

### ISSUE THREAD

**Issue by Zirfern** [Kyrobi/Cynagen#247]
Please title this post "Ban Appeal:" followed by your username.

### Please fill in the information below

**Minecraft username**: __GGCLAN3422

**I was banned for**: inappropriate dms

**A brief summary of what happened before the ban was**: so earlier today i made bad mistakes as i havent gotten any sleep in 2 days almost, do not ask why please and i talked to kei about it cuz i thought she can help me and she got me perma banned and blocked me i did not think i would get perma banned it might have been a bad mistake on my part to talk about it but i did not know i would get perma banned just because of it, i did not say anything very inappropriate i just summed up what happened earlier in the day

**Why is what you got banned for bad?**:
I dont know why tbh i did not say anything mean or anything perma ban worthy i believe 
**I like to be unbanned because**:
This server is great and i didnt think that could get me banned it is a mistake on my part that i will fix now that i understand the mistake of what i did
**Reflect on what you did wrong. What are some things you will start doing to improve yourself?**: many things i need to improve on i am a nice guy but i can get a bit weird at times which has led to mistakes in the past and me falling out of relationships(friendships)  with people and in this situation i did not think that a few things that i said could get me banned but now i understand it could and i am deeply sorry for that mistake i can understand if im banned for like a week or two maybe a month but not lifetime ban and i will not make the same mistake

**Explain why and when the best time to mow the lawn is**: 8 in the morning then u dont have to do it while busy during day

**Additional information**: (Optional)
Very sorry again to Keilius i should have asked if it is ok for me to tell them what happened u wish i didnt as i felt me and her had a decent friendship in the game and she helped me out very often with situations, i thought she helps me with my newest mistake i did which i also feel bad about and this server is my life i have made good friends here i hope not to lose because of dumb decisions on my part, i hope you guys find it in your hearts to un ban me as i will not make the same mistake again deeply sorry to everyone involved in this and whoever had to read this whole thing hope you guys can agree to unban me thanks for reading this

**Comment by Zirfern** [comment]
Sorry about punctuation i dont like to do punctuation, i just like run ons

**Comment by Zirfern** [comment]
Also yes i understand the mistake i made and its a mistake i have had for many years where i say before i think so im working on it i always forget to add stuff jeez

**Comment by Kyrobi** [comment]
hi i took a look at your appeal and I discussion it amongst the staff and the conclusion would be to leave you banned on the discord by reduce your ban on the server after a discussion with the staff it would be a 2 week ban instead of a perm but please note to never dm people innaprotipaite stuff especially if you're a minor and talk about things like that

**Comment by AnotherRevenant** [comment]
TL;DR, you do one more dumb thing you go bye bye

**Comment by Zirfern** [comment]
That is fair rev and im very sorry kyrobi it will not happen again i think now im done i need sleep before i go 70 hours with no sleep
